lib/commands/project/project.js

```diff
@@ -456,6 +457,17 @@ function testGenerate(directory, options, cb) {
             assertionFormat: assertionFormat
           };
 
+          // pass list of paths targeted for load testing
+          if (loadTesting) {
+            if ((typeof loadTesting) !== 'boolean' && fs.existsSync(path.join(directory, loadTesting))) {
+              config.loadTest = (JSON.parse(fs.readFileSync(path.join(directory, loadTesting)))).loadTargets;
```

Comment:
If we're doing this in multiple places, shouldn't we make a helper function for it?

```
JSON.parse(fs.readFileSync(path.join(directory, "FILE NAME")));
```